src/pages/nutrition/tableRow.jsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import TableCell from "@mui/material/TableCell";
+import TableRow from "@mui/material/TableRow";
+import TextField from "@mui/material/TextField";
+import SelectUnit from "./unitSelect";
+import { Typography } from "@mui/material";
+import { Box } from "@mui/material";
+import DeleteOutlineIcon from "@mui/icons-material/DeleteOutline";
+
+import { useTranslation } from "react-i18next";
+
+function createData(label, property, deleteIcon) {
+  return { label, property, deleteIcon };
+}
+
+export default function TableRowComp({
+  nutriment,
+  onchangeHandler,
+  deleteItem,
+}) {
+  const { t } = useTranslation();
+
+  const row = createData(
+    <Box
+      sx={{
+        display: "flex",
+        justifyContent: "space-between",
+        flexDirection: "row",
+        width: "400px",
+      }}
+    >
+      <Typography>{t(`nutrition.nutriments.${nutriment.label}`)}</Typography>
+      <Box
+        sx={{
+          display: "flex",
+        }}
+      >
+        <SelectUnit
+          options={nutriment.quantification}
+          value={"quantification"}
+          onchangeHandler={onchangeHandler}
+        />
```

Comment:
You use two ties `SelectUnit` and not always to select unit

Why not directly put the `Select` here

```suggestion
        <TextField select value={nutriment.quantification}>
        	<MenuItem value="=">=</MenuItem>
  	</TextField>
```

Reply:
The problem was that we have different unit. But now I think it is better to add maybe unitsList property into nutrients data